app/webpack.config.js

```diff
@@ -222,6 +228,26 @@ class CompileSchemasPlugin {
   }
 }
 
+/**
+ * A helper for filtering deprecated webpack loaders, to be replaced with assets
```

Comment:
Maybe the upstream base webpack config should be updated to assets too at some point.